### DISCUSSION TITLE
Avatar

### DISCUSSION THREAD

**Discussion by cslsselu** [community/community#61589] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How do I change my avataar?

**Comment by A-one1** [comment]
Click on your profile picture in the top-right corner of GitHub.
Select "Settings" from the dropdown menu.
On the tab called public profiile, click on the Edit button on the avatar box and choose your own picture

**Comment by queenofcorgis** [comment]
@cslsselu @A-one1 GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
